### DISCUSSION TITLE
Could not insert module cortx-motr/m0tr.ko: Network is down

### DISCUSSION THREAD

**Discussion by hessio** [Seagate/cortx-motr#598] [Q&A] [answered]
```
[root@cortxtest cortx-motr]#  sudo ./scripts/m0 run-ut
----- run_ut -----
insmod: ERROR: could not insert module /root/cortx-motr/m0tr.ko: Network is down
Error loading /root/cortx-motr/m0tr.ko node_uuid=abcdef01-2345-6789-0123-456789ABCDEF Aborting.
Error unloading /root/cortx-motr/m0tr.ko.
```
Does anyone know what could be causing this issue?

**Comment by andriytk** [comment, marked as answer]
Looks like some problem with lnet (you can confirm with `lctl list_nids` and check for errors with `dmesg | tail -50`). If it's the case and you see in dmesg something like:

```
[41061.409598] LNetError: 8891:0:(socklnd.c:2795:ksocknal_startup()) ksocklnd: requested 1 but only 0 interfaces found
[41062.409898] LNetError: 105-4: Error -100 starting up LNI tcp
[41062.411483] subsystem net/lnet init failed: rc = -100
```

Try to restart lnet:

```
sudo rmmod lnet
```

If it refuses to unload because of some dependent module (like ksocklnd) - unload that module also:

```
sudo rmmod ksocklnd lnet
```

And then start lnet back:

```
sudo modprobe lnet
sudo lctl net up
```

**Comment by kiwionly2** [comment]
This info is very helpful, Is that possible to add to Quick start guide [troubleshooting](https://github.com/Seagate/cortx-motr/blob/main/doc/Quick-Start-Guide.rst#Troubleshooting) section ?

**Reply by andriytk** [reply]
Sure, feel free to create a PR for this @kiwionly2.
